### DISCUSSION TITLE
A Few Things Missing on GitHub Profile

### DISCUSSION THREAD

**Discussion by Neodevils** [community/community#35886] [Mobile] [answered]
## Current version of GitHub user profile:

![IMG_20221012_120052](https://user-images.githubusercontent.com/71941576/195303379-a08c7a43-f12d-42f1-8177-4cf5ba0f7077.jpg)

## How should it be, what must be added to profile?

* Timezone
* PRO badge
* Sponsor button
* Achievements
* Organizations

**Comment by iskae** [comment, marked as answer]
Hello @Neodevils 👋 

You are definitely on 🔥 😂 Thanks so much for all these suggestions 🙇 

We already have an internal issue for profile improvements and I will be adding your suggestions in there too 👍 

Cheers!

**Reply by Neodevils** [reply]
Hahaha, thank you! 😂
